packages/api-client/src/connections/CentralApi.ts

```diff
@@ -6,14 +6,20 @@
 import { BaseApi } from './BaseApi';
 
 export type SurveyResponse = {
-  surveyId: string;
-  entityCode: string;
-  timestamp: string;
-  answers: Answers;
+  id?: string;
+  survey_id: string;
+  entity_id?: string;
+  entity_code?: string;
+  timestamp?: string;
+  data_time?: string;
+  start_time?: string;
+  end_time?: string;
+  approval_status?: string;
+  answers: Answers | Answers[];
```

Comment:
Can we remove optional configs?

Reply:
As in flag some of these fields as non-optional? Yup, if they're required we may as well mark them as such 👍